cli/src/pcluster/config/cluster_config.py

```diff
@@ -1030,9 +1030,6 @@ def _register_validators(self):
             SubnetsValidator, subnet_ids=self.compute_subnet_ids + [self.head_node.networking.subnet_id]
         )
         self._register_storage_validators()
-        self._register_validator(
-            HeadNodeLaunchTemplateValidator, head_node=self.head_node, ami_id=self.head_node_ami, tags=self.tags
-        )
```

Comment:
Because in the base class we don't have self.scheduling:
scheduler_tags = self.scheduling.settings.scheduler_definition.tags

Yes, tags are unique are validate through ComputeResourceLaunchTemplateValidator:
"message": "Unable to validate configuration parameters for instance type t2.micro. Please double check your cluster configuration. Duplicate tag key 'key1' specified."
For example, if we have tags in scheduler plugin:
```
      Tags:
        - Key: key1
          Value: value1
```
and Tags in cluster:
```
Tags:
  - Key: key1
    Value: value1
  - Key: key2
    Value: value1
```
It will raise the validation error